cmd/unifibackup/daemon.go

```diff
@@ -8,12 +8,12 @@ import (
 	"sync"
 	"time"
 
-	"github.com/gebn/unifibackup/v2/cmd/unifibackup/monitor"
-	"github.com/gebn/unifibackup/v2/cmd/unifibackup/uploader"
-
```

Comment:
This is interesting, I've always grouped imports by stdlib, project packages, then other packages. Do you have a reference for swapping other and project?

Reply:
oh my editor is setup to auto organize imports, i think via gopls or https://github.com/incu6us/goimports-reviser, i havn't really thought about it as long as its consistent.